101-servicebus-create-namespace/azuredeploy.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serviceBusNamespaceName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Service Bus namespace"
+            }
+        },
+        "serviceBusSku": {
+            "type": "string",
+            "allowedValues": [
+                "Basic",
+                "Standard"
+            ],
+            "defaultValue": "Standard",
+            "metadata": {
+                "description": "The messaging tier for service Bus namespace"
+            }
+        },
+        "serviceBusApiVersion": {
```

Comment:
api version shouldn't be a parameter for any of these templates. It's hard for the user to figure this one out. Also when there is a breaking change to the API version, the template needs to be updated before it can work with the newer api version. Make this a variable instead with hardcoded version.